osu.Android/OsuGameActivity.cs

```diff
@@ -49,6 +52,8 @@ private void handleIntent(Intent intent)
                 case Intent.ActionDefault:
                     if (intent.Scheme == ContentResolver.SchemeContent)
                         handleImportFromUri(intent.Data);
+                    else if (intent.Scheme == osu_url_scheme)
+                        Task.Run(() => game.HandleLink(intent.DataString));
```

Comment:
I'm not sure whether it's really all that required to `Task.Run` a single schedule, but iOS sort of does the same thing for some reason... Was there a reason for this?

Reply:
I looked up commit history in the iOS project as well as the related PRs to find a clue why the link handling was wrapped in a task but couldn't find any information on it , and so I decided to wrap the call, in default of not having any clue why it had been done so in iOS, just in case. But yeah not sure either it is required since it is anyway scheduled further down in HandleLink() method itself so guess it'll cause no harm to directly call HandleLink().